draft-ietf-openpgp-pqc.md

```diff
@@ -1572,23 +1572,23 @@ longer signature generation time.
 
 # IANA Considerations
 
-IANA will add the following registries to the `Pretty Good Privacy (PGP)`
-registry group at https://www.iana.org/assignments/pgp-parameters:
+IANA is requested to add the following registries to the `OpenPGP`
+registry group at https://www.iana.org/assignments/openpgp/openpgp.xhtml:
```

Comment:
i would drop the trailing `/openpgp.xhtml` -- or maybe drop the whole URL, since we want the updates to happen to the registry itself, not just the html variant.

Reply:
> i would drop the trailing `/openpgp.xhtml`

Fixed now.